Client/src/Core/routes.tsx

```diff
@@ -188,17 +189,20 @@ const routes: RouteEntry[] = [
     component: BasketController
   },
 
-  {
-    path: '/workspace/datasets',
-    requiresLogin: true,
-    component: (props: RouteComponentProps<{}>) => {
-      const { location } = props;
-      return <UserDatasetListController location={location}/>
-    }
-  },
+  // {
+  //   path: '/workspace/datasets/new-upload',
+  //   requiresLogin: true,
+  //   component: () => <UserDatasetNewUploadController />
+  // },
+
+  // {
+  //   path: '/workspace/datasets/recent-uploads',
+  //   requiresLogin: true,
+  //   component: () => <UserDatasetAllUploadsController />
+  // },
```

Comment:
Should be fine to delete,
```
path: '/workspace/datasets',
    exact: false,
```
explained to me well enough where to look for a component responsible for these paths